meetings/2023-09-06.md

```diff
@@ -0,0 +1,164 @@
+  * Legal check: [requested review from RW](https://github.com/solid/process/blob/7916591d1ffdc4cfae7cf2c8a11772c198ae7db0/solid-cg-charter.md#license).
+* SC: If there are no rights or licensing incompatibilities or patent policy complications for the CG or the proposed WG, the PR can be merged (including any editorial clarifications) and resolving [MIT License and/or W3C CLA and FSA](https://github.com/solid/process/issues/327).
+* SC: Any other changes to the CG charter will follow the "Amendments to this Charter" process.
+* SC: Relatively minor follow up: I am looking into latest published version URL. Admin stuff: Publishing at w3.org entails copying CG charter into a WordPress page available from the [W3C Solid CG homepage](https://www.w3.org/community/solid/). Roughly half of the CGs with a charter use w3.org and the other half uses github.io. 
+* SC: We'll have some follow-ups next.
+* SC: I'd like TBL to comment on Vision and RW on Legal. 
+* TBL: On licensing, I posted my conclusion on Matrix. If the charter is finalized that is incompatible with you wanting my review. In particular snippets of code you want to paste in your program. We looked at the MIT license but then looked at MDN, and they say that any snippet wherever it is, will be public domain. I think the MDN people had a good idea. That discussion obviously did not make its way into the charter.
+* WT: Thanks TBL for the input. Discussion on the GH issue was about MIT license, which is what was incorporated into the charter, but I think it is definitely possible to add public domain for snippets. 
+* TBL: Happy with MIT license. But CC0 for code snippets in specs or tutorials. 
+* SC: I think it's a work item.
+* TBL: No, part of a work item. Code snippets within a document that could have W3C/MIT license, but snippets are CC0. 
+* SC: This licensing issue is about the work item, you are saying distinguishing snippets. That was one of the suggestions in the issue. 
+* RW: I want to reassure TBL that the only issue I see is the notices. Imagine you have software and need notices. We already have a practice of notices, you can take the code somewhere, but then a multitude of notices. IF we accept that notices should be provided, then the CLA would already do what Tim wants. You can co-license with MIT because CLA says "I grant perpetual license with no obligation to reproduce, distribute, etc.." meaning we have it already. This is rather later for WG, when you want to clearly distinguish the specification with the rules where you don't want someone making derivatives of the spec, but you want the code/pseudo code to be reused as much as possible. Similar issue to test-suite, where you don't want to ??? As it stands, the things TBL wants to accomplish can be done now. From a legal standpoint we are clear. Clarify pseudocode is MIT license. 
```

Comment:
```suggestion
* RW: I want to reassure TBL that the only issue I see is the notices. Imagine you have software and need notices. We already have a practice of notices, you can take the code somewhere, but then a multitude of notices. IF we accept that notices should be provided, then the CLA would already do what Tim wants. You can co-license with MIT because CLA says "I grant perpetual license with no obligation to reproduce, distribute, etc." meaning we have it already. This is rather late for WG, when you want to clearly distinguish the specification with the rules where you don't want someone making derivatives of the spec, but you want the code/pseudocode to be reused as much as possible. Similar issue to test-suite, where you don't want to ??? As it stands, the things TBL wants to accomplish can be done now. From a legal standpoint we are clear. Clarify pseudocode is MIT license. 
```